libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -657,6 +657,13 @@ AP_InertialSensor::AP_InertialSensor() :
     _singleton = this;
     AP_Param::setup_object_defaults(this, var_info);
 
+#if INS_MAX_INSTANCES < 3
```

Comment:
this would introduce a memory overwrite bug, as _use[] array is of length INS_MAX_INSTANCES

Reply:
Fixed